sdk/js/src/relayer/__tests__/wormhole_relayer.ts

```diff
@@ -0,0 +1,690 @@
+      keepSending: true,
+      newMessages: [payload2, "0x00"],
+      chains: [sourceChainId],
+      gasLimits: [500000],
+    };
+    const tx = await sourceMockIntegration.sendMessagesWithFurtherInstructions(
+      [payload1],
+      furtherInstructions,
+      [targetChainId],
+      [value.add(extraForwardingValue)],
+      { value: value.add(extraForwardingValue), gasLimit: 500000 }
+    );
+    console.log("Sent delivery request!");
+    const rx = await tx.wait();
+    console.log("Message confirmed!");
+
+    return tx.hash
+}
+
+describe("Wormhole Relayer Tests", () => {
+
+  test("Executes a Delivery Success", async () => {
+    const arbitraryPayload = ethers.utils.hexlify(
+      ethers.utils.toUtf8Bytes(generateRandomString(32))
+    );
+    console.log(`Sent message: ${arbitraryPayload}`);
+    
+    await testSend(arbitraryPayload);
+
+    await waitForRelay();
+
+    console.log("Checking if message was relayed");
+    const message = await targetMockIntegration.getMessage();
+    console.log(`Sent message: ${arbitraryPayload}`);
+    console.log(`Received message: ${message}`);
+    expect(message).toBe(arbitraryPayload);
+  });
+
+  test("Reads Delivery Success status through SDK", async () => {
+    const arbitraryPayload = ethers.utils.hexlify(
+      ethers.utils.toUtf8Bytes(generateRandomString(32))
+    );
+    console.log(`Sent message: ${arbitraryPayload}`);
+    
+    const txHash = await testSend(arbitraryPayload);
+
+    await waitForRelay();
+
+    const message = await targetMockIntegration.getMessage();
+    console.log(`Sent message: ${arbitraryPayload}`);
+    console.log(`Received message: ${message}`);
+    expect(message).toBe(arbitraryPayload);
+
+    console.log("Checking status using SDK");
+
+    const status = await getStatus(txHash);
+    
+    expect(status).toBe("Delivery Success");
+  });
+
+
+  test("Executes a forward", async () => {
+    const arbitraryPayload1 = ethers.utils.hexlify(
+      ethers.utils.toUtf8Bytes(generateRandomString(32))
+    );
+    const arbitraryPayload2 = ethers.utils.hexlify(
+      ethers.utils.toUtf8Bytes(generateRandomString(32))
+    );
+    console.log(`Sent message: ${arbitraryPayload1}`);
+    
+    const txHash = await testForward(arbitraryPayload1, arbitraryPayload2);
+
+    await waitForRelay(2);
+
+    console.log("Checking if message was relayed");
+    const message1 = await targetMockIntegration.getMessage();
+    console.log(
+      `Sent message: ${arbitraryPayload1} (expecting ${arbitraryPayload2} from forward)`
+    );
+    console.log(`Received message on target: ${message1}`);
+    expect(message1).toBe(arbitraryPayload1);
+
+    console.log("Checking if forward message was relayed back");
+    const message2 = await sourceMockIntegration.getMessage();
+    console.log(`Sent message: ${arbitraryPayload2}`);
+    console.log(`Received message on source: ${message2}`);
+    expect(message2).toBe(arbitraryPayload2);
```

Comment:
numbers in variable names are typically a dead giveaway that the variables in question should instead be an array

(also applies to the rest of the code down below)